app/assets/javascripts/school_administrator_dashboard/discipline_dashboard/SchoolDisciplineDashboard.test.js

```diff
@@ -1,30 +1,113 @@
 import React from 'react';
-import moment from 'moment';
 import {mount} from 'enzyme';
+import moment from 'moment';
+import {toMomentFromTimestamp} from '../../helpers/toMoment';
+import {withNowMoment} from '../../testing/NowContainer';
+import PerDistrictContainer from '../../components/PerDistrictContainer';
 import {
   createStudents,
-  testSchool
+  testSchool,
+  testHighSchool
 } from '../DashboardTestData';
 import SchoolDisciplineDashboard from './SchoolDisciplineDashboard';
-import {withDefaultNowContext} from '../../testing/NowContainer';
 
+jest.mock('react-virtualized'); // doesn't work in test without a real DOM
```

Comment:
FWIW if you want to use snapshot tests on something that uses `react-virtualized` you can do that using a similar approach to the one in https://github.com/studentinsights/studentinsights/pull/2196.  It's fine to not do that here, just sharing about a new way to approach this that I was trying.